pkg/ibmcsidriver/controller_helper.go

```diff
@@ -585,6 +610,11 @@ func createCSIVolumeResponse(vol provider.Volume, volAccessPointResponse provide
 
 	labels[NFSServerPath] = volAccessPointResponse.MountPath
 
+	// Update label in case EIT is enabled
```

Comment:
Can you one more label isENIenabled ? Also if possible lets put security Group and subnet ID labels so that we can make out from the pv describe and avoid looking at logs.

Reply:
Opened issue https://github.com/IBM/ibm-vpc-file-csi-driver/issues/76